static/js/src/advantage/react/components/Subscriptions/SubscriptionList/SubscriptionList.tsx

```diff
@@ -39,6 +40,39 @@ const SubscriptionList = ({ selectedId, onSetActive }: Props) => {
     select: selectBlenderSubscriptions,
   });
 
+  const [showingDirectPurchaseModal, setShowingDirectPurchaseModal] = useState(
```

Comment:
I added this code to the subscription list mostly as a hack. A more appropriate place would probably be the `Content` component, or even the `Subscriptions` component (`static/js/src/advantage/react/components/Subscriptions/Subscriptions.tsx`).